.github/workflows/ci.yml

```diff
@@ -36,6 +36,8 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
+      - name: Add UbuntuGIS ppa
+        run: sudo add-apt-repository ppa:ubuntugis/ppa
```

Comment:
Please add the info about this requirement into our README.md, under Deployment or Dependencies.

Reply:
22fc0234da600237aecb0e4deedfb5bf3d476cd2